eclass/cmake.eclass

```diff
@@ -702,12 +702,18 @@ cmake_src_test() {
 # @DESCRIPTION:
 # Function for installing the package. Automatically detects the build type.
 cmake_src_install() {
+	local generator
+	case ${CMAKE_MAKEFILE_GENERATOR} in
+		ninja) generator="eninja" ;;
+		*) generator="${CMAKE_MAKEFILE_GENERATOR}" ;;
+	esac
```

Comment:
```suggestion
	local generator=${CMAKE_MAKEFILE_GENERATOR}
	case ${CMAKE_MAKEFILE_GENERATOR} in
		ninja) generator="eninja" ;;
	esac
```

Reply:
That is indeed cleaner, I changed it. Thanks!